ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -398,7 +398,15 @@ export class RegistrationDashboard extends React.Component<RegistrationDashboard
              data-test-id='success-message'>
           You successfully completed all the required steps to access the Researcher Workbench.
           <Button style={{marginLeft: '0.5rem'}}
-                  onClick={() => window.location.reload()}>Get Started</Button>
+                  onClick={() => {
+                    // Quirk / hack note: the goal here is to send the user to the homepage once they've completed
+                    // all access modules. Normally we would just navigate(['']) to do this. However, because
+                    // of the way this dashboard is rendered *within* the homepage component, a call to
+                    // navigate is not enough to trigger the normal homepage to load. As a workaround, we
```

Comment:
Does `navigate(['']); window.location.reload()` work or does that revert the url to include the search again?

Reply:
The latter – I tried that first, and unfortunately the window.location.reload() uses the original window.location, apparently before navigate(['']) has any effect on changing it.

Reply:
rats.

Is this related to RW-4915 or just discovered around the same time

Reply:
Unrelated, just discovered around the same time.